pyproject.toml

```diff
@@ -47,6 +47,7 @@ dependencies = [
     "openpyxl",
     "attrs",
     "utm",
+    "pyyaml-include"
```

Comment:
We have the load_yaml functionality in hopp, I don't think this is necessary? It doesn't look like it's used anywhere in this PR

Reply:
we do indeed have an include loader set up for our `load_yaml` function. I have removed pyyaml-include.